fth/fth-bib.sty

```diff
@@ -37,14 +38,18 @@ maxnames=2]{biblatex}
 \DeclareFieldFormat{pages}{#1}	% keine "S."-Angaben
 \DeclareFieldFormat{postnote}{#1}	% keine "S."-Angaben
 \renewcommand{\newunitpunct}{\addcomma\space}
-\renewbibmacro*{finentry}{}
 
 % Autoren bei \citeauthor in Small caps, wenn Option gesetzt
 \ifdefined\@authorsc
   \renewcommand*{\mkbibnamefamily}[1]{\textsc{#1}}
   \renewcommand*{\mkbibnameprefix}[1]{\textsc{#1}}
 \fi
 
+\ifdefined\@finDot
```

Comment:
Könntest du hier noch nen kleine Kommentar dran machen was das tut, damit ich’s in nem Jahr auch noch checke 😅